style.css

```diff
@@ -1,3 +1,315 @@
+* {
+    margin: 0;
+    padding: 0;
+}
+
+
 body {
-    background-color: lightgray;
+    text-align: center;
+}
+
+
+/* Feel free to remove these styles or customise in your own stylesheet 👍  */
+.attribution {
+    font-size: 11px;
+    text-align: center;
+}
+
+.attribution a {
+    color: hsl(228, 45%, 44%);
+}
+
+.backgroundColor {
+    color: rgb (246, 246, 246);
+    width: 0%;
+
+}
+
+
+.mainContainer {
+    display: flex;
+    text-align: center;
+    width: 100%;
+    /* border: 1px;
+    border-style: solid; */
+}
+
+.mainHeading {
+    font-size: 40px;
+    text-align: left;
+}
+
+.mainText {
+    font-size: 16px;
+    color: darkgray;
+    padding-top: 50px;
+    width: 400px;
+    text-align: left;
+    padding-bottom: 50px;
+}
+
+.mainButton {
+    width: 110px;
+    border-radius: 5px;
+    padding: 5px 8px;
+    border: 1px solid black;
+}
+
+.mainButton a {
+    color: black;
+    text-decoration: none;
+    font-weight: bold;
+}
+
+.leftColumn {
+    width: 50%;
+    padding-top: 120px;
+    padding-left: 100px;
+
+}
+
+.rightColumn {
+    width: 50%;
+    padding-top: 50px;
+
+}
+
+
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap");
+
+* {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+
+body {
+    background-color: #FAFAFA;
+    /* color: white; */
+    font-family: "Poppins", sans-serif;
+}
+
+header a {
+    text-decoration: none;
+}
+
+header {
+    padding: 0 20px;
+    background-color: #FAFAFA;
+    height: 50px;
+    display: flex;
+    justify-content: space-between;
+}
+
+#logo {
+    font-weight: bold;
+    font-size: 18px;
+    display: flex;
+    align-items: center;
+    width: 8%;
+    padding-top: 10px;
+}
+
+#logo a {
+    color: black;
+}
+
+#topLinks {
+    display: flex;
+    /* justify-content: left; */
+    align-items: left;
+    width: 80%;
+}
+
+#rightLinks {
+    display: block;
+    /* justify-content: left; */
+    align-items: right;
+    width: 12%;
+    margin-top: 10px;
+}
+
+ul {
+    list-style: none;
+    height: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: space-around;
+}
+
+ul a {
+    color: white;
+}
+
+ul li {
+    padding: 5px;
+    margin-left: 10px;
+}
+
+ul li:hover {
+    transform: scale(1.1);
+    transition: 0.3s;
+}
+
+#login,
+#signup {
+    border-radius: 5px;
+    padding: 5px 8px;
+}
+
+#login {
+    /* border: 1px solid black; */
+}
+
+#signup {
+    border: 1px solid black;
+}
+
+#signup a {
+    color: black;
+}
+
+#login a {
+    color: black;
+}
+
+#hamburger-icon {
+    margin: auto 0;
+    display: none;
+    cursor: pointer;
+}
+
+#hamburger-icon div {
+    width: 35px;
+    height: 3px;
+    background-color: white;
+    margin: 6px 0;
+    transition: 0.4s;
+}
+
+.open .bar1 {
+    -webkit-transform: rotate(-45deg) translate(-6px, 6px);
```

Comment:
There are tools that automatically add all necessary postfixes for various browsers. You shouldn't to add this manually in your code. 
I suggest removing this for now.

A learning area here is preparing and optimising HTML, CSS and JavaScript files for the web using automation. There are tools like Webpack or Parcel that do this for you, but you need to learn how to set it up, terminal basics, etc. This is a big topic that you will eventually get to on your front-end dev roadmap. 

Let me know if you are interested in this topic and I can give more resources.